src/main/java/org/interledger/cryptoconditions/ConditionImpl.java

```diff
@@ -2,17 +2,29 @@
 
 import java.util.EnumSet;
 
+
 import org.interledger.cryptoconditions.Condition;
 import org.interledger.cryptoconditions.ConditionType;
 import org.interledger.cryptoconditions.FeatureSuite;
+import org.interledger.cryptoconditions.encoding.Base64Url;
 
-final class ConditionImpl implements Condition {
+public final class ConditionImpl implements Condition {
```

Comment:
> previously ConditionInputStream was instantiating an anonymous Condition, but it was not a real implementation

Right, but do we need to use the ConditionImpl there? I think we can use the anonymous condition in this case because it really is just a nice model to hang data off.

It makes all the other stuff much simpler.